pkgs/development/python-modules/eth-typing/default.nix

```
@@ -18,20 +19,22 @@ buildPythonPackage rec {
     owner = "ethereum";
     repo = "eth-typing";
     rev = "refs/tags/v${version}";
-    hash = "sha256-JT/2bCPYFSRNt3V7QnHSAJR7HrZ1JpRKdU7gQpoYIn0=";
+    hash = "sha256-WFTx5u85Gp+jQPWS3BTk1Pky07C2fVAzwrG/c3hSRzM=";
   };
 
   nativeBuildInputs = [ setuptools ];
 
   nativeCheckInputs = [ pytestCheckHook ];
 
+  propagatedBuildInputs = [ typing-extensions ];
```

```suggestion
  dependencies = [ typing-extensions ];
```
`dependencies` is the modern alias used for `propagatedBuildInputs`

`nativeBuildInputs = [ setuptools ]` could also be changed to `build-system = [ setuptools ]` in a similar way